### DISCUSSION TITLE
How to highlate generated response text  using flowise tools(different colours,bold text)

### DISCUSSION THREAD

**Discussion by trepont** [FlowiseAI/Flowise#474] [Q&A]
When the generated response is displayed, the highlighted text will be shown with a different background color or font style, and the bold text will be displayed in a bold font.
How achieve the task of displaying highlighted text with a different background color or font style, as well as displaying bold text in a bold font in Flowise AI Chartflow design
![Screenshot_20](https://github.com/FlowiseAI/Flowise/assets/136782819/a05bc475-3778-4eb4-a1e5-1cfe4234b922)

**Comment by HenryHengZJ** [comment]
you can modify the css file here - https://github.com/FlowiseAI/Flowise/blob/main/packages/ui/src/views/chatmessage/ChatMessage.css
